201-aci-linuxcontainer-volume-emptydir/README.md

```diff
@@ -0,0 +1,3 @@
+# Azure Container Instances
+
+This template demonstrates a simple use case for emptyDir volume of [Azure Container Instances](https://docs.microsoft.com/en-us/azure/container-instances/).
```

Comment:
Add the deploy to azure and visualize buttons (see other samples for example (non-aci ones))